tests/Unit/Helpers/IO/VolumeDataHelpers.hpp

```diff
@@ -0,0 +1,181 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <algorithm>
+#include <boost/iterator/transform_iterator.hpp>
+#include <cstddef>
+#include <cstdint>
+#include <optional>
+#include <string>
+#include <vector>
+
+#include "IO/H5/AccessType.hpp"
+#include "IO/H5/File.hpp"
+#include "IO/H5/VolumeData.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+#include "Utilities/Algorithm.hpp"
+#include "Utilities/MakeString.hpp"
+#include "Utilities/Numeric.hpp"
+
+
+namespace TestVolumeData_detail {
```

Comment:
To be consistent: `TestHelpers::io::volume_data` (or `VolumeData` at the end), then document `volume_data` namespace

Reply:
Happy to make this change. By the way, I was just following what OberverHelpers.hpp does when I named this namespace. After looking across several test helpers domains, it looks to me like there isn't always consistency with the namespace names or with documenting them with doxygen

Reply:
Yep, though we are trying to move to the "standard" :) another one of those things that, if I could time travel, I'd go back and start with the convention at the beginning :) Thanks for changing this!